README.md

```diff
@@ -46,6 +46,21 @@ We pass any provided options into the `imagemin` itself:
   ];
 }
 ```
+### WebP Support
```

Comment:
Can we turn this section in "Custom plugins"? And by `webp` plugin we can illustrate how to add custom plugins.

Sounds good?

Reply:
Please, also add some "If you want to add a custom plugin for `imagemin` ..." intro text.